### DISCUSSION TITLE
关于Nginx访问日志时区设置问题

### DISCUSSION THREAD

**Discussion by kekylin** [0xJacky/nginx-ui#410] [Q&A] [answered]
请教一下，通过Docker方式部署的Nginx-UI，我看到Nginx访问日志记录的时间不是本地的时间，请问如何将其设置为本地时间？
部署使用的参数：
`version: "3.3"
services:
  nginx-ui:
    stdin_open: true
    tty: true
    container_name: nginx-ui
    restart: always
    network_mode: host
    environment:
      - TZ=Asia/Shanghai
    volumes:
      - /srv/docker/data/nginx-ui/nginx:/etc/nginx
      - /srv/docker/data/nginx-ui/nginx-ui:/etc/nginx-ui
      - /srv/docker/data/nginx-ui/www:/var/www
    image: uozi/nginx-ui:latest`

实际是北京时间白天访问的，但是日志记录到的访问时间是夜里：
![QQ截图20240531173626](https://github.com/0xJacky/nginx-ui/assets/65521882/9c122721-c771-44b8-98cc-9a212a474558)

**Comment by 0xJacky** [comment]
可能是这个版本的镜像有问题，我空时确认一下

**Comment by 0xJacky** [comment, marked as answer]
进入容器手动执行一下，然后重启容器就可以了，下次发新镜像的时候会一并修复。

```
ln -sf /usr/share/zoneinfo/${TZ} /etc/localtime && \
    echo "${TZ}" > /etc/timezone
```

**Reply by kekylin** [reply]
好的，谢谢！

**Reply by 0xJacky** [reply]
直接用 latest tag 的镜像也可以，已经更新了
